docker/complete/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+MAINTAINER Mesosphere <support@mesosphere.io>
+
+RUN locale-gen en_US.UTF-8
+RUN dpkg-reconfigure locales
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
```

Comment:
why the `upgrade`? seems (a) not necessary, and; (b) to change from a known state/baseline to a moving target